manifests/sasl-sercret.yaml.in

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sasl-secret
+type: Opaque
+data:
+  username: bGlidmlydC1jbGllbnQ=
+  password: Mm1mS1NGNXA=
```

Comment:
You shouldn't hardcode a password. 

First add a "libvirt_password" env variable to hack/config.sh (initializing it to a randomly generated string)

Make hack/build-manifests.sh know about this new env variable for subsitution

Finally use  'password: {{ libvirt_password }}' here.